Mathlib/Algebra/Tropical/BigOperators.lean

```diff
@@ -114,35 +115,36 @@ theorem trop_infᵢ [ConditionallyCompleteLinearOrder R] [Fintype S] (f : S →
 #align trop_infi trop_infᵢ
 
 theorem Multiset.untrop_sum [LinearOrder R] [OrderTop R] (s : Multiset (Tropical R)) :
-    untrop s.Sum = Multiset.inf (s.map untrop) := by
+    untrop s.sum = Multiset.inf (s.map untrop) := by
   induction' s using Multiset.induction with s x IH
   · simp
-  · simpa [← IH]
+  · simp [← IH]
+    rfl
 #align multiset.untrop_sum Multiset.untrop_sum
 
 theorem Finset.untrop_sum' [LinearOrder R] [OrderTop R] (s : Finset S) (f : S → Tropical R) :
     untrop (∑ i in s, f i) = s.inf (untrop ∘ f) := by
-  cases s
-  convert Multiset.untrop_sum _
-  simpa
+  convert Multiset.untrop_sum (s.val.map f)
+  simp
```

Comment:
Non-terminal simp, please replace it with the output of `simp?`.

Reply:
Oh, that's an error. I didn't push the latest version...

BTW, is something like that okay to finish a proof?  
```
simp 
rfl
```

Reply:
It's probably harmless, but I would still prefer the `simp only` version.